core/domain/skill_domain_test.py

```diff
@@ -83,10 +83,20 @@ def _assert_valid_skill_id(self, expected_error_substring, skill_id):
             utils.ValidationError, expected_error_substring):
             skill_domain.Skill.require_valid_skill_id(skill_id)
 
+    def _assert_valid_skill_ids(self, expected_error_substring, skill_ids):
```

Comment:
It's not worth making a new function just for this. Inline it in the test method instead.

Reply:
Fixed.